activesupport/lib/active_support/inflector/methods.rb

```diff
@@ -217,7 +232,10 @@ def tableize(class_name)
     #   classify('calculus')     # => "Calculu"
     def classify(table_name)
       # strip out any leading schema name
-      camelize(singularize(table_name.to_s.sub(/.*\./, "")))
+      table_name = table_name.to_s
+      dot_idx = table_name.index(".")
```

Comment:
this transforms "string.with.multiple.dots" differently than it used to.

Reply:
perhaps should use 'rindex' instead.

Reply:
Worth a test case?